### PR TITLE
New Keyboard: TG4x

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

New Keyboard TG4x. This is a very limited run, should this go into handwired? 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tg4x/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# TG4x v2.1
+
+40% sandwich keyboard driven by a pro micro. 
+
+For more info please refer to [MythosMann's Github](www.github.com/MythosMann/tg4x).  
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)
```
Comment (fauxpark) [inline keyboards/tg4x/readme.md]:
```suggestion
* Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)
```

File: keyboards/tg4x/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# TG4x v2.1
+
+40% sandwich keyboard driven by a pro micro. 
+
+For more info please refer to [MythosMann's Github](www.github.com/MythosMann/tg4x).  
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: TG4x v2.1 PCB
```
Comment (fauxpark) [inline keyboards/tg4x/readme.md]:
```suggestion
* Hardware Supported: TG4x v2.1 PCB
```

File: keyboards/tg4x/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# TG4x v2.1
+
+40% sandwich keyboard driven by a pro micro. 
+
+For more info please refer to [MythosMann's Github](www.github.com/MythosMann/tg4x).  
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: TG4x v2.1 PCB  
+Hardware Availability: [Geekhack](https://geekhack.org/index.php?topic=99163.0)
```
Comment (fauxpark) [inline keyboards/tg4x/readme.md]:
```suggestion
* Hardware Availability: [Geekhack](https://geekhack.org/index.php?topic=99163.0)
```

File: keyboards/tg4x/config.h [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```
Comment (drashna) [inline keyboards/tg4x/config.h]: No product ID change?

**Pull request merged**